charts/opencost/templates/route.yaml

```
@@ -0,0 +1,32 @@
+{{- if and .Values.opencost.ui.enabled .Values.opencost.ui.route.enabled }}
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: {{ include "opencost.fullname" . }}-route
+  namespace: {{ include "opencost.namespace" . }}
+  labels: {{- include "opencost.labels" . | nindent 4 }}
+  {{- with .Values.opencost.ui.route.annotations }}
+  annotations: {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  host: {{ .Values.opencost.ui.route.host | quote }}
+  {{- if .Values.opencost.ui.route.path }}
+  path: {{ .Values.opencost.ui.route.path }}
+  {{- end }}
+  port:
+    targetPort: {{ .Values.opencost.ui.route.targetPort }}
+  to:
+    kind: Service
+    name: opencost
```

```suggestion
    name: {{ include "opencost.fullname" . }}
```
https://github.com/opencost/opencost-helm-chart/blob/c810b378802b24beaae0480f31b97be7beaca88a/charts/opencost/templates/service.yaml#L5

---

Fixed.